packages/server/src/course/ical.service.ts

```diff
@@ -44,8 +44,8 @@ export class IcalService {
     );
 
     const isOfficeHoursEvent = (title: string) => {
-      const nonOfficeHourKeywords = ['Lecture', 'Lab', 'Exam', 'Class'];
-      return nonOfficeHourKeywords.every(keyword => !title.includes(keyword));
+      const offficeHourKeywords = ['OH', 'Hours'];
```

Comment:
Note: The "Hours" is the keyword that is used in Networks (CS 3700) to distinguish office hours